molecule/cookiecutter/collection/{{cookiecutter.collection_name}}/galaxy.yml

```diff
@@ -0,0 +1,38 @@
+##################################################
```

Comment:
JFYI you can use https://pytest-cookies.readthedocs.io/ to test cookiecutter template rendering.